kits/trust/contracts/TrustKit.sol

```diff
@@ -0,0 +1,342 @@
+        acl.createPermission(tokenManager, voting, voting.CREATE_VOTES_ROLE(), voting);
+        acl.createPermission(voting, voting, voting.MODIFY_QUORUM_ROLE(), voting);
+        acl.createPermission(voting, voting, voting.MODIFY_SUPPORT_ROLE(), voting);
+    }
+
+    function _createTokenManagerPermissions(ACL acl, TokenManager tokenManager, Voting voting) internal {
+        acl.createPermission(address(this), tokenManager, tokenManager.MINT_ROLE(), address(this));
+        acl.createPermission(voting, tokenManager, tokenManager.ASSIGN_ROLE(), voting);
+        acl.createPermission(voting, tokenManager, tokenManager.REVOKE_VESTINGS_ROLE(), voting);
+    }
+
+    function _createEVMRegistryPermissions(ACL acl, Voting holdVoting) internal {
+        EVMScriptRegistry evmScriptsRegistry = EVMScriptRegistry(acl.getEVMScriptRegistry());
+        bytes32 addExecutorRole = evmScriptsRegistry.REGISTRY_ADD_EXECUTOR_ROLE();
+        acl.createPermission(holdVoting, evmScriptsRegistry, addExecutorRole, holdVoting);
+        bytes32 managerRole = evmScriptsRegistry.REGISTRY_MANAGER_ROLE();
+        acl.createPermission(holdVoting, evmScriptsRegistry, managerRole, holdVoting);
+    }
+
+    function _createFinancePermissions(ACL acl, Voting holdVoting, Finance finance, Vault vault) internal {
+        acl.createPermission(finance, vault, vault.TRANSFER_ROLE(), holdVoting);
+        acl.createPermission(holdVoting, finance, finance.CREATE_PAYMENTS_ROLE(), holdVoting);
+        acl.createPermission(holdVoting, finance, finance.EXECUTE_PAYMENTS_ROLE(), holdVoting);
+        acl.createPermission(holdVoting, finance, finance.MANAGE_PAYMENTS_ROLE(), holdVoting);
+    }
+
+    function _mintHoldTokens(TokenManager holdTokenManager, address[] beneficiaryKeys) internal {
+        holdTokenManager.mint(beneficiaryKeys[0], 1e18);
+        holdTokenManager.mint(beneficiaryKeys[1], 1e18);
+    }
+
+    function _mintHeirsTokens(TokenManager heirsTokenManager, address[] heirs, uint256[] heirsStake, uint256 blockedHeirsSupply) internal {
+        heirsTokenManager.mint(address(0), blockedHeirsSupply);
+        for (uint256 i = 0; i < heirs.length; i++) {
+            heirsTokenManager.mint(heirs[i], heirsStake[i]);
+        }
+    }
+
+    function _createMultiSigPermissions(Kernel dao, ACL acl, MultiSigWallet multiSig, Voting holdVoting, TokenManager holdTokenManager, TokenManager heirsTokenManager)
+        internal
+    {
+        acl.createPermission(multiSig, holdTokenManager, holdTokenManager.BURN_ROLE(), multiSig);
+        acl.createPermission(multiSig, heirsTokenManager, heirsTokenManager.BURN_ROLE(), multiSig);
+        cleanupPermission(acl, multiSig, holdTokenManager, holdTokenManager.MINT_ROLE());
+        cleanupPermission(acl, multiSig, heirsTokenManager, heirsTokenManager.MINT_ROLE());
+        cleanupPermission(acl, holdVoting, dao, dao.APP_MANAGER_ROLE());
```

Comment:
No big deal, but it seems this one and the next one don't match with function's name.